frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf

```diff
@@ -119,7 +119,7 @@
                     '<div title="<sw360:out value='${vulnerabilityVerificationTooltips.get(vulnerability.externalId).get(releaseId)}'/>"><sw360:DisplayEnum value="${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId)}"/></div>',
                 </core_rt:otherwise>
                 </core_rt:choose>
-                "5": '<div class="dataTables_cell_nowrap">${vulnerability.action}</div>'
+                "6": '<div class="dataTables_cell_nowrap"<sw360:out value="${vulnerability.action}"/></div>'
```

Comment:
Why you changed this value to 6?
The vulnerability value is after the div close.

Reply:
fixed